api/datadoghq/common/datadogpodautoscaler_types.go

```diff
@@ -274,23 +274,27 @@ const (
 // +kubebuilder:object:generate=true
 type DatadogPodAutoscalerHorizontalStatus struct {
 	// Target is the current target of the horizontal scaling
-	Target *DatadogPodAutoscalerHorizontalTargetStatus `json:"target,omitempty"`
+	Target *DatadogPodAutoscalerHorizontalRecommendation `json:"target,omitempty"`
 
 	// LastActions are the last successful actions done by the controller
 	LastActions []DatadogPodAutoscalerHorizontalAction `json:"lastActions,omitempty"`
+
+	// LastRecommendations stores the most recent recommendations
+	LastRecommendations []DatadogPodAutoscalerHorizontalRecommendation `json:"lastRecommendations,omitempty"`
 }
 
-// DatadogPodAutoscalerHorizontalTargetStatus defines the current target of the horizontal scaling
+// DatadogPodAutoscalerHorizontalRecommendation defines a horizontal scaling recommendation
 // +kubebuilder:object:generate=true
-type DatadogPodAutoscalerHorizontalTargetStatus struct {
+type DatadogPodAutoscalerHorizontalRecommendation struct {
 	// Source is the source of the value used to scale the target workload
-	Source DatadogPodAutoscalerValueSource `json:"source"`
+	// +optional
+	Source DatadogPodAutoscalerValueSource `json:"source,omitempty"`
 
 	// GeneratedAt is the timestamp at which the recommendation was generated
 	GeneratedAt metav1.Time `json:"generatedAt,omitempty"`
 
-	// Replicas is the desired number of replicas for the workload
-	Replicas int32 `json:"desiredReplicas"`
+	// Replicas is the recommended number of replicas for the workload
+	Replicas int32 `json:"replicas"`
```

Comment:
Note: we're going to loose state on upgrade by doing so, it might be easier to keep the name until next major version